src/plugins/workspace/public/components/workspace_form/use_workspace_form.ts

```diff
@@ -89,8 +89,19 @@ export const useWorkspaceForm = ({
     [setFeatureConfigs]
   );
 
+  const getSubmitFormData = (submitFormData: WorkspaceFormDataState) => {
+    return {
+      name: submitFormData.name!,
+      description: submitFormData.description,
+      color: submitFormData.color || '#FFFFFF',
+      features: submitFormData.features,
+      permissionSettings: submitFormData.permissionSettings as WorkspacePermissionSetting[],
+      selectedDataSourceConnections: submitFormData.selectedDataSourceConnections,
+    };
+  };
```

Comment:
looks like the return value is same as WorkspaceFormDataState? can we use `WorkspaceFormDataState` diectly? or `{...WorkspaceFormDataState}`

Reply:
No, there are some differences between the type of return values and WorkspaceFormDataState, such as the useCase attribute.